app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/service/TrackerBlockingVpnService.kt

```diff
@@ -664,10 +666,18 @@ class TrackerBlockingVpnService : VpnService(), CoroutineScope by MainScope(), V
             }
         }
 
-        startForeground(
-            VPN_FOREGROUND_SERVICE_ID,
-            VpnEnabledNotificationBuilder.buildVpnEnabledNotification(applicationContext, vpnNotification),
-        )
+        if (appBuildConfig.sdkInt >= Build.VERSION_CODES.Q) {
```

Comment:
Shouldn't the condition be `appBuildConfig.sdkInt >= Build.VERSION_CODES.TIRAMISU`?

Reply:
@anikiki seems like this API is available for Q and above.
![Screenshot 2024-01-30 at 14 14 18](https://github.com/duckduckgo/Android/assets/2943941/b5d86d72-d760-4f29-9fb3-d2d1fccf24dd)